tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -158,7 +158,7 @@ jobs:
       workingFolder: '$(Build.BinariesDirectory)'
 
   - script: |
-     python -m pip install -q pyopenssl setuptools wheel numpy     
+     python -m pip install -q pyopenssl setuptools wheel numpy==1.18
```

Comment:
Do you also need to update C api packaging pipeline and other pipelines?

Reply:
They don't matter.  They can use any numpy version.

Reply:
We all need to make sure the numpy version specified in "azure-pipelines-py-packaging.yml" matches with the number in "/setup.py"

Reply:
In future It might still make sense to move every pipeline to the same numpy version to avoid any confusion. Current change looks good for now.